lib/plugins/aws/provider.js

```diff
@@ -1199,6 +1199,13 @@ class AwsProvider {
             vpc: { $ref: '#/definitions/awsLambdaVpcConfig' },
             vpcEndpointIds: { $ref: '#/definitions/awsCfArrayInstruction' },
             versionFunctions: { $ref: '#/definitions/awsLambdaVersioning' },
+            websockets: {
```

Comment:
Let's use `provider.websocket` instead of `provider.websockets` to be consistent with the event name